google-client-loader.html

```diff
@@ -112,12 +112,25 @@
         errorEventName: {
           type: String,
           value: 'google-api-load-error'
+        },
+
+        _clientLoaded: {
```

Comment:
these should both be `readOnly: true` and use the respective hidden setter to set the values.